syntax = "proto3";

service sdcs{
    rpc search_kv(request) returns (response){}
    rpc search_kv_local(request) returns (response){}

    rpc update_kv(request) returns (response){}
    rpc update_kv_local(request) returns (response){}

    rpc delete_kv(request) returns (response){}
    rpc delete_kv_local(request) returns (response){}
}

message request{
    string key = 1;
}

message response{
    string value = 1;
}